.github/workflows/api-ci.yml

```diff
@@ -47,6 +47,21 @@ jobs:
       - name: Build everything
         run: pnpm build
 
+      - name: Check for uncommitted database migrations
```

Comment:
added this check to avoid this problem: https://recalllabs.slack.com/archives/C08TV9K3XS5/p1749678705705319